src/ResourceManager/DataFactories/Commands.DataFactoryV2.Test/ScenarioTests/RunTests.ps1

```diff
@@ -43,7 +43,8 @@ function Test-Run
         Set-AzureRmDataFactoryV2Pipeline -ResourceGroupName $rgname -Name $pipelineName -DataFactoryName $dfname -File ".\Resources\pipeline.json" -Force
 
         $Run = Invoke-AzureRmDataFactoryV2Pipeline -ResourceGroupName $rgname -PipelineName $pipelineName -DataFactoryName $dfname -Parameter @{"OutputBlobName"="test";}
-        
+        Assert-True { Stop-AzureRmDataFactoryV2PipelineRun -ResourceGroupName $rgname -PipelineRunId $Run -DataFactoryName $dfname -Force -PassThru}
+
```

Comment:
There is a bigger problem here that none of the Get cmdlets are not being thoroughly tested, but this is good enough for the Stop cmdlet.

Reply:
Yes, I saw that too. The author probably wanted to just make sure it doesn't throw an exception. I will follow up with them.